content/projects/data-science-specific/bootcamp-exploratory-data-analysis/_index.md

```diff
@@ -0,0 +1,55 @@
+---
+content_type: project
+flavours:
+- python
+prerequisites:
+  hard:
+  - topics/data-science-specific/datacamp/intro-to-python
+  soft:
+  - topics/data-science-specific/intro-to-google-colab
+ready: true
+submission_type: repo
+title: Bootcamp Exploratory Data Analysis
+---
+
+You will need to download and add all the relevant files from [here](https://drive.google.com/drive/folders/1scmLfz6SswLqKoj1S6s3VytezCR7m_dJ?usp=sharing) and add them to the repo supplied.
+
+Your repo should contain everything needed to replicate your work:
+- data
+- notebook
+- `requirements.txt`
+
+## Instructions
+
+Your task is to use the given data to answer the questions found in `questions.md`. Any other additional analysis you think will help your submission is cool with us but NO external data may be used.
+
+We will not only be assessing your code but also how you structure and present your analysis.
+
+## Setting Up Your Environment
```

Comment:
I'm not sure tat Setting Up Your Environment should be here. It doesn't seem DRY.

See here: content/environment-setup/_index.md 
If the existing environment setup instructions are insufficient then those should  rather be updated.

Reply:
You're write this isn't necessary.